diff -Naur openssh-8.1p1-old/channels.c openssh-8.1p1-new/channels.c
--- openssh-8.1p1-old/channels.c	2019-10-08 17:31:03.000000000 -0700
+++ openssh-8.1p1-new/channels.c	2019-12-18 13:57:14.147000015 -0800
@@ -62,6 +62,7 @@
 # include <stdint.h>
 #endif
 #include <stdio.h>
+#include <stddef.h>
 #include <stdlib.h>
 #include <string.h>
 #include <termios.h>
@@ -4687,6 +4688,8 @@
 {
 	int sock;
 	struct sockaddr_un addr;
+	int abstract = 0;
+	int addrsize = 0;
 
 	sock = socket(AF_UNIX, SOCK_STREAM, 0);
 	if (sock == -1)
@@ -4694,8 +4697,15 @@
 	memset(&addr, 0, sizeof(addr));
 	addr.sun_family = AF_UNIX;
 	strlcpy(addr.sun_path, pathname, sizeof addr.sun_path);
-	if (connect(sock, (struct sockaddr *)&addr, sizeof(addr)) == 0)
+	addrsize = offsetof(struct sockaddr_un, sun_path) + strlen(addr.sun_path);
+	if (addr.sun_path[0] = '@')
+		abstract = 1;
+	if (abstract)
+		addr.sun_path[0] = '\0';
+	if (connect(sock, (struct sockaddr *)&addr, addrsize) == 0)
 		return sock;
+	if (abstract)
+		addr.sun_path[0] = '@';
 	close(sock);
 	error("connect %.100s: %.100s", addr.sun_path, strerror(errno));
 	return -1;
diff -Naur openssh-8.1p1-old/defines.h openssh-8.1p1-new/defines.h
--- openssh-8.1p1-old/defines.h	2019-10-08 17:31:03.000000000 -0700
+++ openssh-8.1p1-new/defines.h	2019-12-18 13:56:03.459000015 -0800
@@ -450,7 +450,7 @@
 #  ifdef __hpux
 #    define X_UNIX_PATH "/var/spool/sockets/X11/%u"
 #  else
-#    define X_UNIX_PATH "/tmp/.X11-unix/X%u"
+#    define X_UNIX_PATH "@/tmp/.X11-unix/X%u"
 #  endif
 #endif /* X_UNIX_PATH */
 #define _PATH_UNIX_X X_UNIX_PATH
