diff -Naur mythtv-master-20191130-a27754a-old/mythtv/libs/libmythui/mythuibuttonlist.cpp mythtv-master-20191130-a27754a-new/mythtv/libs/libmythui/mythuibuttonlist.cpp
--- mythtv-master-20191130-a27754a-old/mythtv/libs/libmythui/mythuibuttonlist.cpp	2019-12-23 22:20:15.000000000 -0800
+++ mythtv-master-20191130-a27754a-new/mythtv/libs/libmythui/mythuibuttonlist.cpp	2019-12-23 22:20:45.609999984 -0800
@@ -2639,6 +2639,12 @@
         {
             ShowSearchDialog();
         }
+        else if (action == "0" || action == "1" || action == "2" || action == "3" ||
+                 action == "4" || action == "5" || action == "6" || action == "7" ||
+                 action == "8" || action == "9")
+        {
+            SetItemCurrent((int) ((GetCount() * (action.toInt() / 10.0))));
+        }
         else
             handled = false;
     }
