diff -Naur mythtv-0.30-20191130-a27754a-old/mythtv/configure mythtv-0.30-20191130-a27754a-new/mythtv/configure
--- mythtv-0.30-20191130-a27754a-old/mythtv/configure	2019-12-29 11:48:35.000000000 -0800
+++ mythtv-0.30-20191130-a27754a-new/mythtv/configure	2019-12-30 14:56:10.911998212 -0800
@@ -6514,7 +6514,7 @@
 EOF
 fi
 
-enabled qtwebkit || die "Error! QtWebkit headers not found"
+enabled qtwebkit || warn "Warning! QtWebkit headers not found"
 
 if $(pkg-config --exists Qt5Script) || $(pkg-config --exists QtScript) ; then
     enable qtscript
