diff -Naur binutils-2.34-old/bfd/configure binutils-2.34-new/bfd/configure
--- binutils-2.34-old/bfd/configure	2020-02-01 03:49:13.000000000 -0800
+++ binutils-2.34-new/bfd/configure	2020-03-16 17:52:52.664935923 -0700
@@ -10646,7 +10646,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/binutils/configure binutils-2.34-new/binutils/configure
--- binutils-2.34-old/binutils/configure	2020-02-01 03:50:05.000000000 -0800
+++ binutils-2.34-new/binutils/configure	2020-03-16 17:52:52.665935923 -0700
@@ -10445,7 +10445,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/gas/configure binutils-2.34-new/gas/configure
--- binutils-2.34-old/gas/configure	2020-02-01 03:49:51.000000000 -0800
+++ binutils-2.34-new/gas/configure	2020-03-16 17:52:52.666935923 -0700
@@ -10261,7 +10261,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/gprof/configure binutils-2.34-new/gprof/configure
--- binutils-2.34-old/gprof/configure	2020-02-01 03:52:21.000000000 -0800
+++ binutils-2.34-new/gprof/configure	2020-03-16 17:52:52.668935923 -0700
@@ -10108,7 +10108,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/ld/configure binutils-2.34-new/ld/configure
--- binutils-2.34-old/ld/configure	2020-02-01 03:50:25.000000000 -0800
+++ binutils-2.34-new/ld/configure	2020-03-16 17:52:52.669935923 -0700
@@ -10950,7 +10950,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/libctf/configure binutils-2.34-new/libctf/configure
--- binutils-2.34-old/libctf/configure	2020-01-18 06:02:31.000000000 -0800
+++ binutils-2.34-new/libctf/configure	2020-03-16 18:00:07.612912213 -0700
@@ -10349,7 +10349,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir=../lib
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/libiberty/Makefile.in binutils-2.34-new/libiberty/Makefile.in
--- binutils-2.34-old/libiberty/Makefile.in	2020-01-18 05:55:48.000000000 -0800
+++ binutils-2.34-new/libiberty/Makefile.in	2020-03-16 17:52:52.669935923 -0700
@@ -365,7 +365,7 @@
 # multilib-specific flags, it's overridden by FLAGS_TO_PASS from the
 # default multilib, so we have to take CFLAGS into account as well,
 # since it will be passed the multilib flags.
-MULTIOSDIR = `$(CC) $(CFLAGS) -print-multi-os-directory`
+MULTIOSDIR = '../lib'
 install_to_libdir: all
 	if test -n "${target_header_dir}"; then \
 		${mkinstalldirs} $(DESTDIR)$(libdir)/$(MULTIOSDIR); \
diff -Naur binutils-2.34-old/libtool.m4 binutils-2.34-new/libtool.m4
--- binutils-2.34-old/libtool.m4	2020-01-18 05:55:49.000000000 -0800
+++ binutils-2.34-new/libtool.m4	2020-03-16 17:52:52.670935923 -0700
@@ -2056,7 +2056,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/opcodes/configure binutils-2.34-new/opcodes/configure
--- binutils-2.34-old/opcodes/configure	2020-02-01 03:49:42.000000000 -0800
+++ binutils-2.34-new/opcodes/configure	2020-03-16 17:52:52.671935923 -0700
@@ -10357,7 +10357,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.34-old/zlib/configure binutils-2.34-new/zlib/configure
--- binutils-2.34-old/zlib/configure	2020-01-18 05:55:49.000000000 -0800
+++ binutils-2.34-new/zlib/configure	2020-03-16 17:58:23.852917869 -0700
@@ -9544,7 +9544,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir=../lib 2>/dev/null`
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
@@ -11438,8 +11438,8 @@
   toolexecdir='$(libdir)/gcc-lib/$(target_alias)'
   toolexeclibdir='$(libdir)'
 fi
-if test "$GCC" = yes && $CC -print-multi-os-directory > /dev/null 2>&1; then
-  multiosdir=/`$CC -print-multi-os-directory`
+if test "$GCC" = yes && ../lib > /dev/null 2>&1; then
+  multiosdir=/../lib
   case $multiosdir in
     /.) multiosdir= ;; # Avoid trailing /.
   esac
diff -Naur binutils-2.34-old/zlib/configure.ac binutils-2.34-new/zlib/configure.ac
--- binutils-2.34-old/zlib/configure.ac	2020-01-18 05:55:49.000000000 -0800
+++ binutils-2.34-new/zlib/configure.ac	2020-03-16 17:57:22.144921233 -0700
@@ -100,8 +100,8 @@
   toolexecdir='$(libdir)/gcc-lib/$(target_alias)'
   toolexeclibdir='$(libdir)'
 fi
-if test "$GCC" = yes && $CC -print-multi-os-directory > /dev/null 2>&1; then
-  multiosdir=/`$CC -print-multi-os-directory`
+if test "$GCC" = yes && ../lib > /dev/null 2>&1; then
+  multiosdir=/../lib
   case $multiosdir in
     /.) multiosdir= ;; # Avoid trailing /.
   esac
