diff -Naur iguanaIR-1.1.0-old/daemon.c iguanaIR-1.1.0-new/daemon.c
--- iguanaIR-1.1.0-old/daemon.c	2013-07-28 10:08:08.000000000 -0700
+++ iguanaIR-1.1.0-new/daemon.c	2020-03-27 08:11:55.085027687 -0700
@@ -21,6 +21,7 @@
 #include <signal.h>
 #include <string.h>
 #include <errno.h>
+#include <sys/select.h>
 #include <sys/stat.h>
 
 #include <argp.h>
diff -Naur iguanaIR-1.1.0-old/iguanaIR.c iguanaIR-1.1.0-new/iguanaIR.c
--- iguanaIR-1.1.0-old/iguanaIR.c	2013-07-22 10:09:45.000000000 -0700
+++ iguanaIR-1.1.0-new/iguanaIR.c	2020-03-27 08:13:58.228020974 -0700
@@ -17,6 +17,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <errno.h>
+#include <limits.h>
 
 #include "pipes.h"
 #include "support.h"
diff -Naur iguanaIR-1.1.0-old/pipes.c iguanaIR-1.1.0-new/pipes.c
--- iguanaIR-1.1.0-old/pipes.c	2013-07-22 10:09:53.000000000 -0700
+++ iguanaIR-1.1.0-new/pipes.c	2020-03-27 08:08:54.219037546 -0700
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <errno.h>
+#include <sys/select.h>
 #include <sys/stat.h>
 
 #include "pipes.h"
diff -Naur iguanaIR-1.1.0-old/server.c iguanaIR-1.1.0-new/server.c
--- iguanaIR-1.1.0-old/server.c	2013-07-22 10:09:53.000000000 -0700
+++ iguanaIR-1.1.0-new/server.c	2020-03-27 08:08:54.219037546 -0700
@@ -11,6 +11,7 @@
  * See LICENSE for license details.
  */
 
+#include <limits.h>
 #include <malloc.h>
 
 #include "iguanaIR.h"
