diff -Naur gettext-0.20.2-old/gettext-tools/misc/gettextize.in gettext-0.20.2-new/gettext-tools/misc/gettextize.in
--- gettext-0.20.2-old/gettext-tools/misc/gettextize.in	2020-04-13 18:32:21.000000000 -0700
+++ gettext-0.20.2-new/gettext-tools/misc/gettextize.in	2020-04-14 11:33:41.550590354 -0700
@@ -1303,10 +1303,10 @@
     5) count="five paragraphs";;
     *) count="$count paragraphs";;
   esac
-  echo "Press Return to acknowledge the previous $count."
-  # Read from /dev/tty, not stdin, so that gettextize cannot be abused by
-  # non-interactive tools.
-  read dummy < /dev/tty
+#  echo "Press Return to acknowledge the previous $count."
+#  # Read from /dev/tty, not stdin, so that gettextize cannot be abused by
+#  # non-interactive tools.
+#  read dummy < /dev/tty
 fi
 
 exit 0
