diff -Naur gettext-0.20.1-old/gettext-tools/misc/gettextize.in gettext-0.20.1-new/gettext-tools/misc/gettextize.in
--- gettext-0.20.1-old/gettext-tools/misc/gettextize.in	2019-05-11 14:22:34.000000000 -0700
+++ gettext-0.20.1-new/gettext-tools/misc/gettextize.in	2019-11-19 14:52:28.874003307 -0800
@@ -1303,10 +1303,10 @@
     5) count="five paragraphs";;
     *) count="$count paragraphs";;
   esac
-  echo "Press Return to acknowledge the previous $count."
-  # Read from /dev/tty, not stdin, so that gettextize cannot be abused by
-  # non-interactive tools.
-  read dummy < /dev/tty
+#  echo "Press Return to acknowledge the previous $count."
+#  # Read from /dev/tty, not stdin, so that gettextize cannot be abused by
+#  # non-interactive tools.
+#  read dummy < /dev/tty
 fi
 
 exit 0
