diff -Naur evtest-evtest-1.34/evtest.c evtest-648f5c1a/evtest.c
--- evtest-evtest-1.34/evtest.c	2019-08-02 11:14:30.000000000 -0700
+++ evtest-648f5c1a/evtest.c	2019-11-30 12:02:39.000000000 -0800
@@ -56,9 +56,16 @@
 #include <getopt.h>
 #include <ctype.h>
 #include <signal.h>
+#include <limits.h>
 #include <sys/time.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <limits.h> /* PATH_MAX */
+
+#ifndef input_event_sec
+#define input_event_sec time.tv_sec
+#define input_event_usec time.tv_usec
+#endif
 
 #define BITS_PER_LONG (sizeof(long) * 8)
 #define NBITS(x) ((((x)-1)/BITS_PER_LONG)+1)
@@ -1139,7 +1146,7 @@
 			type = ev[i].type;
 			code = ev[i].code;
 
-			printf("Event: time %ld.%06ld, ", ev[i].time.tv_sec, ev[i].time.tv_usec);
+			printf("Event: time %ld.%06ld, ", ev[i].input_event_sec, ev[i].input_event_usec);
 
 			if (type == EV_SYN) {
 				if (code == SYN_MT_REPORT)
