diff -Naur xwiimote-2-old/lib/core.c xwiimote-2-new/lib/core.c
--- xwiimote-2-old/lib/core.c	2013-06-15 05:49:37.212000000 -0700
+++ xwiimote-2-new/lib/core.c	2020-05-19 09:52:40.139075865 -0700
@@ -812,7 +812,8 @@
 	}
 
 	memset(ev, 0, sizeof(*ev));
-	memcpy(&ev->time, &input.time, sizeof(struct timeval));
+	ev->time.tv_sec = input.input_event_sec;
+	ev->time.tv_usec = input.input_event_usec;
 	ev->type = XWII_EVENT_KEY;
 	ev->v.key.code = key;
 	ev->v.key.state = input.value;
@@ -842,7 +843,8 @@
 
 	if (input.type == EV_SYN) {
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 		memcpy(ev->v.abs, &dev->accel_cache, sizeof(dev->accel_cache));
 		ev->type = XWII_EVENT_ACCEL;
 		return 0;
@@ -884,7 +886,8 @@
 
 	if (input.type == EV_SYN) {
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 		memcpy(&ev->v.abs, dev->ir_cache, sizeof(dev->ir_cache));
 		ev->type = XWII_EVENT_IR;
 		return 0;
@@ -936,7 +939,8 @@
 
 	if (input.type == EV_SYN) {
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 
 		ev->v.abs[0].x = dev->mp_cache.x - dev->mp_normalizer.x / 100;
 		ev->v.abs[0].y = dev->mp_cache.y - dev->mp_normalizer.y / 100;
@@ -988,7 +992,8 @@
 
 	if (input.type == EV_SYN) {
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 		memcpy(&ev->v.abs, dev->bboard_cache,
 		       sizeof(dev->bboard_cache));
 		ev->type = XWII_EVENT_BALANCE_BOARD;
@@ -1117,7 +1122,8 @@
 		}
 
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 		ev->type = XWII_EVENT_PRO_CONTROLLER_KEY;
 		ev->v.key.code = key;
 		ev->v.key.state = input.value;
@@ -1133,7 +1139,8 @@
 			dev->pro_cache[1].y = input.value;
 	} else if (input.type == EV_SYN) {
 		memset(ev, 0, sizeof(*ev));
-		memcpy(&ev->time, &input.time, sizeof(struct timeval));
+		ev->time.tv_sec = input.input_event_sec;
+		ev->time.tv_usec = input.input_event_usec;
 		memcpy(&ev->v.abs, dev->pro_cache,
 		       sizeof(dev->pro_cache));
 		ev->type = XWII_EVENT_PRO_CONTROLLER_MOVE;
