diff -Naur xorg-server-1.20.7-old/hw/xfree86/common/xf86pciBus.c xorg-server-1.20.7-new/hw/xfree86/common/xf86pciBus.c
--- xorg-server-1.20.7-old/hw/xfree86/common/xf86pciBus.c	2020-01-31 10:10:53.286890894 -0800
+++ xorg-server-1.20.7-new/hw/xfree86/common/xf86pciBus.c	2020-01-31 10:11:45.645891406 -0800
@@ -1106,7 +1106,7 @@
         driverList[0] = "ast";
         break;
     case 0x1002:
-        driverList[0] = "ati";
+        driverList[0] = "radeon";
         break;
     case 0x102c:
         driverList[0] = "chips";
