diff -Naur ntp-4.2.8p13-old/ntpdc/Makefile.am ntp-4.2.8p13-new/ntpdc/Makefile.am
--- ntp-4.2.8p13-old/ntpdc/Makefile.am	2019-12-29 11:13:12.335999975 -0800
+++ ntp-4.2.8p13-new/ntpdc/Makefile.am	2019-12-29 11:15:20.970002841 -0800
@@ -126,7 +126,7 @@
 ntpdc-layout.o: nl.c
 
 layout.here: ntpdc-layout
-	./ntpdc-layout > $@
+	ntp-ntpdc-layout > $@
 
 check-layout: ntpdc-layout $(srcdir)/layout.std layout.here
 	cmp $(srcdir)/layout.std layout.here && echo stamp > $@
diff -Naur ntp-4.2.8p13-old/ntpdc/Makefile.in ntp-4.2.8p13-new/ntpdc/Makefile.in
--- ntp-4.2.8p13-old/ntpdc/Makefile.in	2019-12-29 11:13:12.334999975 -0800
+++ ntp-4.2.8p13-new/ntpdc/Makefile.in	2019-12-29 11:15:20.971002752 -0800
@@ -1205,7 +1205,7 @@
 ntpdc-layout.o: nl.c
 
 layout.here: ntpdc-layout
-	./ntpdc-layout > $@
+	ntp-ntpdc-layout > $@
 
 check-layout: ntpdc-layout $(srcdir)/layout.std layout.here
 	cmp $(srcdir)/layout.std layout.here && echo stamp > $@
