diff -Naur gcc-9.2.0-old/libstdc++-v3/config.h.in gcc-9.2.0-new/libstdc++-v3/config.h.in
--- gcc-9.2.0-old/libstdc++-v3/config.h.in	2019-11-19 10:17:19.968000007 -0800
+++ gcc-9.2.0-new/libstdc++-v3/config.h.in	2019-11-19 10:24:18.012058621 -0800
@@ -151,7 +151,7 @@
 #undef HAVE_FCNTL_H
 
 /* Define to 1 if you have the <fenv.h> header file. */
-#undef HAVE_FENV_H
+#define HAVE_FENV_H 1
 
 /* Define to 1 if you have the `finite' function. */
 #undef HAVE_FINITE
@@ -205,7 +205,7 @@
 #undef HAVE_HYPOTL
 
 /* Define if you have the iconv() function. */
-#undef HAVE_ICONV
+#define HAVE_ICONV 1
 
 /* Define to 1 if you have the <ieeefp.h> header file. */
 #undef HAVE_IEEEFP_H
@@ -244,7 +244,7 @@
 #undef HAVE_ISWBLANK
 
 /* Define if LC_MESSAGES is available in <locale.h>. */
-#undef HAVE_LC_MESSAGES
+#define HAVE_LC_MESSAGES 1
 
 /* Define to 1 if you have the `ldexpf' function. */
 #undef HAVE_LDEXPF
@@ -364,7 +364,7 @@
 #undef HAVE_READLINK
 
 /* Define to 1 if you have the `setenv' function. */
-#undef HAVE_SETENV
+#define HAVE_SETENV 1
 
 /* Define to 1 if you have the `sincos' function. */
 #undef HAVE_SINCOS
@@ -412,7 +412,7 @@
 #undef HAVE_STDLIB_H
 
 /* Define if strerror_l is available in <string.h>. */
-#undef HAVE_STRERROR_L
+#define HAVE_STRERROR_L 1
 
 /* Define if strerror_r is available in <string.h>. */
 #undef HAVE_STRERROR_R
@@ -449,7 +449,7 @@
 #undef HAVE_SYS_IOCTL_H
 
 /* Define to 1 if you have the <sys/ipc.h> header file. */
-#undef HAVE_SYS_IPC_H
+#define HAVE_SYS_IPC_H 1
 
 /* Define to 1 if you have the <sys/isa_defs.h> header file. */
 #undef HAVE_SYS_ISA_DEFS_H
@@ -461,13 +461,13 @@
 #undef HAVE_SYS_PARAM_H
 
 /* Define to 1 if you have the <sys/resource.h> header file. */
-#undef HAVE_SYS_RESOURCE_H
+#define HAVE_SYS_RESOURCE_H 1
 
 /* Define to 1 if you have a suitable <sys/sdt.h> header file */
 #undef HAVE_SYS_SDT_H
 
 /* Define to 1 if you have the <sys/sem.h> header file. */
-#undef HAVE_SYS_SEM_H
+#define HAVE_SYS_SEM_H 1
 
 /* Define to 1 if you have the <sys/socket.h> header file. */
 #undef HAVE_SYS_SOCKET_H
@@ -482,7 +482,7 @@
 #undef HAVE_SYS_SYSINFO_H
 
 /* Define to 1 if you have the <sys/time.h> header file. */
-#undef HAVE_SYS_TIME_H
+#define HAVE_SYS_TIME_H 1
 
 /* Define to 1 if you have the <sys/types.h> header file. */
 #undef HAVE_SYS_TYPES_H
@@ -515,7 +515,7 @@
 #undef HAVE_TIMESPEC_GET
 
 /* Define to 1 if the target supports thread-local storage. */
-#undef HAVE_TLS
+#define HAVE_TLS 1
 
 /* Define if truncate is available in <unistd.h>. */
 #undef HAVE_TRUNCATE
@@ -755,7 +755,7 @@
 #undef HAVE___CXA_THREAD_ATEXIT_IMPL
 
 /* Define as const if the declaration of iconv() needs const. */
-#undef ICONV_CONST
+#define ICONV_CONST 1
 
 /* Define to the sub-directory in which libtool stores uninstalled libraries.
    */
