diff -Naur gcc-9.2.0-old/gcc/config/i386/linux64.h gcc-9.2.0-new/gcc/config/i386/linux64.h
--- gcc-9.2.0-old/gcc/config/i386/linux64.h	2019-01-01 04:31:55.000000000 -0800
+++ gcc-9.2.0-new/gcc/config/i386/linux64.h	2019-11-19 10:19:47.495996594 -0800
@@ -28,8 +28,8 @@
 #define GNU_USER_LINK_EMULATIONX32 "elf32_x86_64"
 
 #define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
-#define GLIBC_DYNAMIC_LINKERX32 "/libx32/ld-linux-x32.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/lib/ld-linux-x86-64.so.2"
+#define GLIBC_DYNAMIC_LINKERX32 "/lib/ld-linux-x32.so.2"
 
 #undef MUSL_DYNAMIC_LINKER32
 #define MUSL_DYNAMIC_LINKER32 "/lib/ld-musl-i386.so.1"
diff -Naur gcc-9.2.0-old/gcc/config/mips/linux.h gcc-9.2.0-new/gcc/config/mips/linux.h
--- gcc-9.2.0-old/gcc/config/mips/linux.h	2019-01-01 04:31:55.000000000 -0800
+++ gcc-9.2.0-new/gcc/config/mips/linux.h	2019-11-19 10:20:57.021995253 -0800
@@ -24,9 +24,9 @@
 #define GLIBC_DYNAMIC_LINKER32 \
   "%{mnan=2008:/lib/ld-linux-mipsn8.so.1;:/lib/ld.so.1}"
 #define GLIBC_DYNAMIC_LINKER64 \
-  "%{mnan=2008:/lib64/ld-linux-mipsn8.so.1;:/lib64/ld.so.1}"
+  "%{mnan=2008:/lib/ld-linux-mipsn8.so.1;:/lib/ld.so.1}"
 #define GLIBC_DYNAMIC_LINKERN32 \
-  "%{mnan=2008:/lib32/ld-linux-mipsn8.so.1;:/lib32/ld.so.1}"
+  "%{mnan=2008:/lib/ld-linux-mipsn8.so.1;:/lib/ld.so.1}"
 
 #undef UCLIBC_DYNAMIC_LINKER32
 #define UCLIBC_DYNAMIC_LINKER32 \
@@ -35,7 +35,7 @@
 #define UCLIBC_DYNAMIC_LINKER64 \
   "%{mnan=2008:/lib/ld64-uClibc-mipsn8.so.0;:/lib/ld64-uClibc.so.0}"
 #define UCLIBC_DYNAMIC_LINKERN32 \
-  "%{mnan=2008:/lib32/ld-uClibc-mipsn8.so.0;:/lib32/ld-uClibc.so.0}"
+  "%{mnan=2008:/lib/ld-uClibc-mipsn8.so.0;:/lib/ld-uClibc.so.0}"
 
 #undef MUSL_DYNAMIC_LINKER32
 #define MUSL_DYNAMIC_LINKER32 \
diff -Naur gcc-9.2.0-old/gcc/config/rs6000/linux64.h gcc-9.2.0-new/gcc/config/rs6000/linux64.h
--- gcc-9.2.0-old/gcc/config/rs6000/linux64.h	2019-07-31 22:57:12.000000000 -0700
+++ gcc-9.2.0-new/gcc/config/rs6000/linux64.h	2019-11-19 10:21:33.530999993 -0800
@@ -427,11 +427,11 @@
 
 #ifdef LINUX64_DEFAULT_ABI_ELFv2
 #define GLIBC_DYNAMIC_LINKER64 \
-"%{mabi=elfv1:%(dynamic_linker_prefix)/lib64/ld64.so.1;" \
+"%{mabi=elfv1:%(dynamic_linker_prefix)/lib/ld64.so.1;" \
 ":%(dynamic_linker_prefix)/lib64/ld64.so.2}"
 #else
 #define GLIBC_DYNAMIC_LINKER64 \
-"%{mabi=elfv2:%(dynamic_linker_prefix)/lib64/ld64.so.2;" \
+"%{mabi=elfv2:%(dynamic_linker_prefix)/lib/ld64.so.2;" \
 ":%(dynamic_linker_prefix)/lib64/ld64.so.1}"
 #endif
 
diff -Naur gcc-9.2.0-old/gcc/config/sparc/linux64.h gcc-9.2.0-new/gcc/config/sparc/linux64.h
--- gcc-9.2.0-old/gcc/config/sparc/linux64.h	2019-04-02 00:40:30.000000000 -0700
+++ gcc-9.2.0-new/gcc/config/sparc/linux64.h	2019-11-19 10:21:49.988977891 -0800
@@ -85,7 +85,7 @@
    done.  */
 
 #define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
-#define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux.so.2"
+#define GLIBC_DYNAMIC_LINKER64 "/lib/ld-linux.so.2"
 
 #ifdef SPARC_BI_ARCH
 
diff -Naur gcc-9.2.0-old/gcc/config/tilegx/linux.h gcc-9.2.0-new/gcc/config/tilegx/linux.h
--- gcc-9.2.0-old/gcc/config/tilegx/linux.h	2019-01-01 04:31:55.000000000 -0800
+++ gcc-9.2.0-new/gcc/config/tilegx/linux.h	2019-11-19 10:22:06.972001591 -0800
@@ -32,7 +32,7 @@
     %{!static: \
       %{rdynamic:-export-dynamic} \
       -dynamic-linker \
-        %{ m32: /lib32/ld.so.1} \
+        %{ m32: /li2/ld.so.1} \
         %{!m32: /lib/ld.so.1}} \
     %{static:-static}}"
 
