diff -Naur mythtv-master-20200409-490e0b1-old/mythtv/configure mythtv-master-20200409-490e0b1-new/mythtv/configure
--- mythtv-master-20200409-490e0b1-old/mythtv/configure	2020-04-09 16:22:02.000000000 -0700
+++ mythtv-master-20200409-490e0b1-new/mythtv/configure	2020-04-09 16:36:32.155149107 -0700
@@ -7008,8 +7008,8 @@
         disable_bindings_perl "Net::UPnP::QueryResponse"
     check_pl_lib "Net::UPnP::ControlPoint"  ||
         disable_bindings_perl "Net::UPnP::ControlPoint"
-    check_pl_lib "IO::Socket::INET6"  ||
-        disable_bindings_perl "IO::Socket::INET6"
+#    check_pl_lib "IO::Socket::INET6"  ||
+#        disable_bindings_perl "IO::Socket::INET6"
     check_pl_lib "XML::Simple"  || disable_bindings_perl "XML::Simple"
 fi
 
