diff -Naur mythtv-master-20200409-490e0b1-old/mythtv/programs/mythfrontend/main.cpp mythtv-master-20200409-490e0b1-new/mythtv/programs/mythfrontend/main.cpp
--- mythtv-master-20200409-490e0b1-old/mythtv/programs/mythfrontend/main.cpp	2020-04-09 16:36:35.325148934 -0700
+++ mythtv-master-20200409-490e0b1-new/mythtv/programs/mythfrontend/main.cpp	2020-04-09 16:36:36.877148850 -0700
@@ -2100,6 +2100,18 @@
                    .arg(port));
         }
     }
+    else
+    {
+        int port = gCoreContext->GetNumSetting("NetworkControlPort", 6546);
+        networkControl = new NetworkControl();
+        QList<QHostAddress> LocalhostList;
+        LocalhostList.clear();
+        LocalhostList.append(QHostAddress("127.0.0.1"));
+        if (!networkControl->listen(LocalhostList, port))
+            LOG(VB_GENERAL, LOG_ERR,
+                QString("NetworkControl failed to bind to port %1.")
+                   .arg(port));
+    }
 
 #if CONFIG_DARWIN
     GetMythMainWindow()->SetEffectsEnabled(false);
