diff -Naur mythtv-master-20191230-bafc80d-old/mythtv/configure mythtv-master-20191230-bafc80d-new/mythtv/configure
--- mythtv-master-20191230-bafc80d-old/mythtv/configure	2019-12-31 15:15:23.000000000 -0800
+++ mythtv-master-20191230-bafc80d-new/mythtv/configure	2019-12-31 15:30:41.798996994 -0800
@@ -6486,17 +6486,17 @@
 esac
 
 enable enforce_wshadow
-case $target_os in
-    android)
-        disable enforce_wshadow
-        ;;
-    linux)
-        . /etc/os-release
-        if test $ID = "centos"; then
-            disable enforce_wshadow
-        fi
-        ;;
-esac
+#case $target_os in
+#    android)
+#        disable enforce_wshadow
+#        ;;
+#    linux)
+#        . /etc/os-release
+#        if test $ID = "centos"; then
+#            disable enforce_wshadow
+#        fi
+#        ;;
+#esac
 
 if $(pkg-config --exists Qt5WebKit) || $(pkg-config --exists QtWebKit) ; then
     enable qtwebkit
