diff -Naur mythtv-master-20200409-490e0b1-old/mythtv/configure mythtv-master-20200409-490e0b1-new/mythtv/configure
--- mythtv-master-20200409-490e0b1-old/mythtv/configure	2020-04-09 16:36:39.984148680 -0700
+++ mythtv-master-20200409-490e0b1-new/mythtv/configure	2020-04-09 16:36:41.570148594 -0700
@@ -6554,17 +6554,17 @@
                 }
 
 enable enforce_wshadow
-case $target_os in
-    android)
-        disable enforce_wshadow
-        ;;
-    linux)
-        . /etc/os-release
-        if test $ID = "centos" && test $VERSION_ID -le 7; then
-            disable enforce_wshadow
-        fi
-        ;;
-esac
+#case $target_os in
+#    android)
+#        disable enforce_wshadow
+#        ;;
+#    linux)
+#        . /etc/os-release
+#        if test $ID = "centos" && test $VERSION_ID -le 7; then
+#            disable enforce_wshadow
+#        fi
+#        ;;
+#esac
 
 if $(pkg-config --exists Qt5WebKit) || $(pkg-config --exists QtWebKit) ; then
     enable qtwebkit
