diff -Naur mythtv-master-20191230-bafc80d-old/mythtv/libs/libmythui/mythuibuttonlist.cpp mythtv-master-20191230-bafc80d-new/mythtv/libs/libmythui/mythuibuttonlist.cpp
--- mythtv-master-20191230-bafc80d-old/mythtv/libs/libmythui/mythuibuttonlist.cpp	2019-12-30 16:02:53.000000000 -0800
+++ mythtv-master-20191230-bafc80d-new/mythtv/libs/libmythui/mythuibuttonlist.cpp	2019-12-30 16:03:25.823999975 -0800
@@ -2618,6 +2618,12 @@
         {
             ShowSearchDialog();
         }
+        else if (action == "0" || action == "1" || action == "2" || action == "3" ||
+                 action == "4" || action == "5" || action == "6" || action == "7" ||
+                 action == "8" || action == "9")
+        {
+            SetItemCurrent((int) ((GetCount() * (action.toInt() / 10.0))));
+        }
         else
             handled = false;
     }
