diff -Naur mythtv-master-20191230-bafc80d-old/mythtv/libs/libmythmetadata/globals.cpp mythtv-master-20191230-bafc80d-new/mythtv/libs/libmythmetadata/globals.cpp
--- mythtv-master-20191230-bafc80d-old/mythtv/libs/libmythmetadata/globals.cpp	2019-12-30 16:03:31.844999975 -0800
+++ mythtv-master-20191230-bafc80d-new/mythtv/libs/libmythmetadata/globals.cpp	2019-12-30 16:03:33.261999975 -0800
@@ -44,5 +44,5 @@
 #ifdef Q_OS_MAC
 const QString DEFAULT_VIDEOSTARTUP_DIR = QDir::homePath() + "/Movies";
 #else
-const QString DEFAULT_VIDEOSTARTUP_DIR = "/share/Movies/dvd";
+const QString DEFAULT_VIDEOSTARTUP_DIR = "/mnt/video";
 #endif
diff -Naur mythtv-master-20191230-bafc80d-old/mythtv/programs/mythfrontend/globalsettings.cpp mythtv-master-20191230-bafc80d-new/mythtv/programs/mythfrontend/globalsettings.cpp
--- mythtv-master-20191230-bafc80d-old/mythtv/programs/mythfrontend/globalsettings.cpp	2019-12-30 16:03:31.914999975 -0800
+++ mythtv-master-20191230-bafc80d-new/mythtv/programs/mythfrontend/globalsettings.cpp	2019-12-30 16:03:33.262999973 -0800
@@ -1406,7 +1406,7 @@
     gc->addSelection(PlaybackSettings::tr("Record date"), "Date");
     gc->addSelection(PlaybackSettings::tr("Season/Episode"), "Season");
     gc->addSelection(PlaybackSettings::tr("Original air date"), "OrigAirDate");
-    gc->addSelection(PlaybackSettings::tr("Program ID"), "Id");
+    gc->addSelection(PlaybackSettings::tr("Program ID"), "Id", true);
 
     gc->setHelpText(PlaybackSettings::tr("Selects how to sort a show's "
                                          "episodes"));
@@ -1709,7 +1709,7 @@
     gc->setLabel(PlaybackSettings::tr("Action on playback exit"));
 
     gc->addSelection(PlaybackSettings::tr("Just exit"), "0");
-    gc->addSelection(PlaybackSettings::tr("Save position and exit"), "2");
+    gc->addSelection(PlaybackSettings::tr("Save position and exit"), "2", true);
     gc->addSelection(PlaybackSettings::tr("Always prompt (excluding Live TV)"),
                      "1");
     gc->addSelection(PlaybackSettings::tr("Always prompt (including Live TV)"),
@@ -1731,7 +1731,7 @@
 
     gc->setLabel(PlaybackSettings::tr("Prompt at end of recording"));
 
-    gc->setValue(false);
+    gc->setValue(true);
 
     gc->setHelpText(PlaybackSettings::tr("If enabled, a menu will be displayed "
                                          "allowing you to delete the recording "
@@ -1829,7 +1829,7 @@
 //
 //     gc->setLabel(PlaybackSettings::tr("Display live preview of recordings"));
 //
-//     gc->setValue(true);
+//     gc->setValue(false);
 //
 //     gc->setHelpText(PlaybackSettings::tr("If enabled, a preview of the recording "
 //                     "will play in a small window on the \"Watch a "
