diff -Naur cmake-3.17.1-old/bootstrap cmake-3.17.1-new/bootstrap
--- cmake-3.17.1-old/bootstrap	2020-04-09 05:33:17.000000000 -0700
+++ cmake-3.17.1-new/bootstrap	2020-04-14 11:09:27.491669618 -0700
@@ -863,6 +863,8 @@
     cmake_bootstrap_system_libs="${cmake_bootstrap_system_libs} -DCMAKE_USE_SYSTEM_LIBRARY_`cmake_toupper $lib`=0" ;;
   --qt-gui) cmake_bootstrap_qt_gui="1" ;;
   --no-qt-gui) cmake_bootstrap_qt_gui="0" ;;
+  --openssl) cmake_bootstrap_system_libs="${cmake_bootstrap_system_libs} -DCMAKE_USE_OPENSSL=1" ;;
+  --no-openssl) cmake_bootstrap_system_libs="${cmake_bootstrap_system_libs} -DCMAKE_USE_OPENSSL=0" ;;
   --qt-qmake=*) cmake_bootstrap_qt_qmake=`cmake_arg "$1"` ;;
   --sphinx-info) cmake_sphinx_info="1" ;;
   --sphinx-man) cmake_sphinx_man="1" ;;
