diff -Naur xf86-input-evdev-2.10.6-old/configure.ac xf86-input-evdev-2.10.6-new/configure.ac
--- xf86-input-evdev-2.10.6-old/configure.ac	2018-05-28 19:33:18.000000000 -0700
+++ xf86-input-evdev-2.10.6-new/configure.ac	2019-12-25 16:09:59.232009977 -0800
@@ -68,7 +68,8 @@
 
 # X Server SDK location is required to install evdev header files
 # This location is also relayed in the xorg-evdev.pc file
-sdkdir=`$PKG_CONFIG --variable=sdkdir xorg-server`
+m4_pattern_allow([^PKG_CONFIG_SYSROOT_DIR$])
+sdkdir="$PKG_CONFIG_SYSROOT_DIR$($PKG_CONFIG --variable=sdkdir xorg-server)"
 
 # Workaround overriding sdkdir to be able to create a tarball when user has no
 # write permission in sdkdir. See DISTCHECK_CONFIGURE_FLAGS in Makefile.am
