diff -Naur binutils-2.33.1-old/bfd/configure binutils-2.33.1-new/bfd/configure
--- binutils-2.33.1-old/bfd/configure	2019-10-12 06:58:20.000000000 -0700
+++ binutils-2.33.1-new/bfd/configure	2019-11-20 11:14:24.566988561 -0800
@@ -10646,7 +10646,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/binutils/configure binutils-2.33.1-new/binutils/configure
--- binutils-2.33.1-old/binutils/configure	2019-10-12 06:59:23.000000000 -0700
+++ binutils-2.33.1-new/binutils/configure	2019-11-20 11:14:24.568988927 -0800
@@ -10441,7 +10441,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/gas/configure binutils-2.33.1-new/gas/configure
--- binutils-2.33.1-old/gas/configure	2019-10-12 06:58:51.000000000 -0700
+++ binutils-2.33.1-new/gas/configure	2019-11-20 11:14:24.570989279 -0800
@@ -10261,7 +10261,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/gprof/configure binutils-2.33.1-new/gprof/configure
--- binutils-2.33.1-old/gprof/configure	2019-10-12 07:05:23.000000000 -0700
+++ binutils-2.33.1-new/gprof/configure	2019-11-20 11:14:24.572989621 -0800
@@ -10108,7 +10108,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/ld/configure binutils-2.33.1-new/ld/configure
--- binutils-2.33.1-old/ld/configure	2019-10-12 06:59:42.000000000 -0700
+++ binutils-2.33.1-new/ld/configure	2019-11-20 11:14:24.573989789 -0800
@@ -10945,7 +10945,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/libiberty/Makefile.in binutils-2.33.1-new/libiberty/Makefile.in
--- binutils-2.33.1-old/libiberty/Makefile.in	2019-09-09 06:19:45.000000000 -0700
+++ binutils-2.33.1-new/libiberty/Makefile.in	2019-11-20 11:14:24.575990116 -0800
@@ -364,7 +364,7 @@
 # multilib-specific flags, it's overridden by FLAGS_TO_PASS from the
 # default multilib, so we have to take CFLAGS into account as well,
 # since it will be passed the multilib flags.
-MULTIOSDIR = `$(CC) $(CFLAGS) -print-multi-os-directory`
+MULTIOSDIR = '../lib'
 install_to_libdir: all
 	if test -n "${target_header_dir}"; then \
 		${mkinstalldirs} $(DESTDIR)$(libdir)/$(MULTIOSDIR); \
diff -Naur binutils-2.33.1-old/libtool.m4 binutils-2.33.1-new/libtool.m4
--- binutils-2.33.1-old/libtool.m4	2019-09-09 06:19:45.000000000 -0700
+++ binutils-2.33.1-new/libtool.m4	2019-11-20 11:14:24.575990118 -0800
@@ -2056,7 +2056,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
diff -Naur binutils-2.33.1-old/opcodes/configure binutils-2.33.1-new/opcodes/configure
--- binutils-2.33.1-old/opcodes/configure	2019-10-12 06:58:44.000000000 -0700
+++ binutils-2.33.1-new/opcodes/configure	2019-11-20 11:14:24.578990587 -0800
@@ -10357,7 +10357,7 @@
   # Ok, now we have the path, separated by spaces, we can step through it
   # and add multilib dir if necessary.
   lt_tmp_lt_search_path_spec=
-  lt_multi_os_dir=`$CC $CPPFLAGS $CFLAGS $LDFLAGS -print-multi-os-directory 2>/dev/null`
+  lt_multi_os_dir='../lib'
   for lt_sys_path in $lt_search_path_spec; do
     if test -d "$lt_sys_path/$lt_multi_os_dir"; then
       lt_tmp_lt_search_path_spec="$lt_tmp_lt_search_path_spec $lt_sys_path/$lt_multi_os_dir"
