diff -Naur nspr-4.24-old/nspr/config/nspr-config.in nspr-4.24-new/nspr/config/nspr-config.in
--- nspr-4.24-old/nspr/config/nspr-config.in	2019-12-02 09:58:37.000000000 -0800
+++ nspr-4.24-new/nspr/config/nspr-config.in	2019-12-15 23:10:13.279006849 -0800
@@ -99,10 +99,10 @@
     exec_prefix=@exec_prefix@
 fi
 if test -z "$includedir"; then
-    includedir=@includedir@
+    includedir=@DESTDIR@@includedir@
 fi
 if test -z "$libdir"; then
-    libdir=@libdir@
+    libdir=@DESTDIR@@libdir@
 fi
 
 if test "$echo_prefix" = "yes"; then
diff -Naur nspr-4.24-old/nspr/configure nspr-4.24-new/nspr/configure
--- nspr-4.24-old/nspr/configure	2019-12-02 09:58:37.000000000 -0800
+++ nspr-4.24-new/nspr/configure	2019-12-15 23:27:37.129987604 -0800
@@ -2507,7 +2507,7 @@
 OBJDIR='$(OBJDIR_NAME)'
 OBJDIR_NAME=.
 OBJDIR_SUFFIX=OBJ
-NSINSTALL='$(MOD_DEPTH)/config/$(OBJDIR_NAME)/nsinstall'
+NSINSTALL='nsinstall'
 NOSUCHFILE=/no-such-file
 LIBNSPR='-L$(dist_libdir) -lnspr$(MOD_MAJOR_VERSION)'
 LIBPLC='-L$(dist_libdir) -lplc$(MOD_MAJOR_VERSION)'
diff -Naur nspr-4.24-old/nspr/configure.in nspr-4.24-new/nspr/configure.in
--- nspr-4.24-old/nspr/configure.in	2019-12-02 09:58:37.000000000 -0800
+++ nspr-4.24-new/nspr/configure.in	2019-12-15 23:27:02.033999469 -0800
@@ -36,7 +36,7 @@
 OBJDIR='$(OBJDIR_NAME)'
 OBJDIR_NAME=.
 OBJDIR_SUFFIX=OBJ
-NSINSTALL='$(MOD_DEPTH)/config/$(OBJDIR_NAME)/nsinstall'
+NSINSTALL=nsinstall
 NOSUCHFILE=/no-such-file
 LIBNSPR='-L$(dist_libdir) -lnspr$(MOD_MAJOR_VERSION)'
 LIBPLC='-L$(dist_libdir) -lplc$(MOD_MAJOR_VERSION)'
