diff -Naur SDL2_ttf-2.0.15-old/configure.in SDL2_ttf-2.0.15-new/configure.in
--- SDL2_ttf-2.0.15-old/configure.in	2019-01-12 11:44:11.000000000 -0800
+++ SDL2_ttf-2.0.15-new/configure.in	2020-04-03 19:53:28.298764111 -0700
@@ -1,6 +1,8 @@
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT(README.txt)
 
+AC_CONFIG_MACRO_DIR([acinclude])
+
 dnl Set various version strings - taken gratefully from the GTk sources
 
 # Making releases:
diff -Naur SDL2_ttf-2.0.15-old/Makefile.am SDL2_ttf-2.0.15-new/Makefile.am
--- SDL2_ttf-2.0.15-old/Makefile.am	2019-01-12 11:44:11.000000000 -0800
+++ SDL2_ttf-2.0.15-new/Makefile.am	2020-04-03 19:53:28.298764111 -0700
@@ -1,5 +1,7 @@
 # Makefile.am for the SDL truetype font loading library and viewer
 
+ACLOCAL_AMFLAGS = -I acinclude
+
 lib_LTLIBRARIES = libSDL2_ttf.la
 
 libSDL2_ttfincludedir = $(includedir)/SDL2
