diff -Naur mame-mame0220-old/makefile mame-mame0220-new/makefile
--- mame-mame0220-old/makefile	2020-04-07 20:37:08.177781877 -0700
+++ mame-mame0220-new/makefile	2020-04-07 20:38:12.871778350 -0700
@@ -563,6 +563,14 @@
 LD := $(OVERRIDE_LD)
 endif
 endif
+ifdef OVERRIDE_AR
+PARAMS += --AR='$(OVERRIDE_AR)'
+ifndef CROSS_BUILD
+AR := $(OVERRIDE_AR)
+endif
+endif
+
+#-------------------------------------------------
 
 #-------------------------------------------------
 # sanity check the configuration
diff -Naur mame-mame0220-old/scripts/genie.lua mame-mame0220-new/scripts/genie.lua
--- mame-mame0220-old/scripts/genie.lua	2020-04-07 20:36:59.057782374 -0700
+++ mame-mame0220-new/scripts/genie.lua	2020-04-07 20:38:12.871778350 -0700
@@ -195,6 +195,11 @@
 }
 
 newoption {
+	trigger = "AR",
+	description = "AR replacement",
+}
+
+newoption {
 	trigger = "TOOLCHAIN",
 	description = "Toolchain prefix"
 }
diff -Naur mame-mame0220-old/scripts/toolchain.lua mame-mame0220-new/scripts/toolchain.lua
--- mame-mame0220-old/scripts/toolchain.lua	2020-04-07 20:36:59.062782374 -0700
+++ mame-mame0220-new/scripts/toolchain.lua	2020-04-07 20:38:12.904778349 -0700
@@ -488,6 +488,9 @@
 	if (_OPTIONS["LD"] ~= nil) then
 		premake.gcc.ld  = _OPTIONS["LD"]
 	end
+	if (_OPTIONS["AR"] ~= nil) then
+		premake.gcc.ar  = _OPTIONS["AR"]
+	end
 
 	configuration {} -- reset configuration
 
