diff -Naur mame-mame0220-old/src/lib/netlist/plib/pexception.cpp mame-mame0220-new/src/lib/netlist/plib/pexception.cpp
--- mame-mame0220-old/src/lib/netlist/plib/pexception.cpp	2020-04-05 10:42:38.000000000 -0700
+++ mame-mame0220-new/src/lib/netlist/plib/pexception.cpp	2020-05-18 12:04:25.743354768 -0700
@@ -7,7 +7,7 @@
 #include <cfenv>
 #include <iostream>
 
-#if (defined(__x86_64__) || defined(__i386__)) && defined(__linux__)
+#if (defined(__x86_64__) || defined(__i386__)) && defined(__linux__) && defined(__GLIBC__)
 #define HAS_FEENABLE_EXCEPT     (1)
 #else
 #define HAS_FEENABLE_EXCEPT     (0)
