diff -Naur libirman-0.4.6-old/configure.ac libirman-0.4.6-new/configure.ac
--- libirman-0.4.6-old/configure.ac	2015-08-18 09:37:04.000000000 -0700
+++ libirman-0.4.6-new/configure.ac	2019-11-22 13:32:16.303000137 -0800
@@ -2,7 +2,7 @@
 AC_INIT([libirman],[0.4.6])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR(irio.c)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE
 
 dnl Checks for programs.
