diff -Naur mesa-18.2.8-old/src/intel/Makefile.am mesa-18.2.8-new/src/intel/Makefile.am
--- mesa-18.2.8-old/src/intel/Makefile.am	2018-12-27 08:02:02.000000000 -0800
+++ mesa-18.2.8-new/src/intel/Makefile.am	2019-12-27 14:16:31.095000560 -0800
@@ -83,7 +83,7 @@
 include Makefile.dev.am
 include Makefile.genxml.am
 include Makefile.isl.am
-include Makefile.tools.am
+#include Makefile.tools.am
 
 include Makefile.vulkan.am
 
